subt_ros/msg/RunStatistics.msg

```diff
@@ -0,0 +1,32 @@
+# This message contains competition statistics
+
+time timestamp
+subt_ros/Robot[] robots
+# subt_ros/Marsupial[] marsupials
```

Comment:
Will marsupials be added?

Reply:
Added: b6713d3c0755763e5ba4d7d0542e09db6aee7dda